arbitrum-docs/node-running/running-an-archive-node.mdx

```diff
@@ -15,9 +15,17 @@ An archival node is a full (nitro) node that also builds an archive of historica
 
 ❗️Note: The minimum storage requirements will change over time as the Nitro chain grows. It is recommended to use more than the minimum requirements to run a robust archival node.
 
+### What Tyoe Of Node You Need to Run For Archive Arbitrum One?
+
+Before the Nitro upgrade happened, Arbitrum One was running on Classic stack for about one year (before block height 2207817).
+Although we have moved Classic's latest state as Nitro's genesis state (see [state migration](../migration/state-migration.mdx)), the Nitro stack can't serve archive requests for pre-nitro blocks. So you have three options to run an Arbitrum One in an archive mode:
+1. If you **only** need to send post-nitro archive requests, you can see [running-a-node](./running-a-node.md).
+2. If you **only** need to send pre-nitro archive requests, you can see [running-a-classic-node](./running-a-classic-node.mdx).
+3. If you need **both** post-nitro and pre-nitro archive requests, you should run **both** an archive Nitro node **and** an archive classic node, there we  explain how to do this.
+
 ### Required Artifacts
```

Comment:
all headers from here should be one hierarchy below `### What Tyoe Of Node You Need to Run For Archive Arbitrum One?` to create the right content hierarchy

so either the previous two are changed to `##` or all of the following ones should be `####`